hubble-be/src/main/resources/i18n/messages_zh_CN.properties

```diff
@@ -102,14 +106,18 @@ execute.predication.error=执行判断语句出错
 reflect.new-instance.failed=通过反射构造类 {0} 的实体失败
 reflect.access-field.failed=通过反射访问类 {1} 的域 {0} 失败
 
+license.verify.graph-connection.failed.preifx=部分图无法继续访问
+license.verify.graph-connection.failed.suffix=请删除后重试
 license.install.failed=安装证书失败
 license.public-cert.invalid=非法的公钥文件
 license.read.public-cert.failed=读取公钥信息失败
 license.read.failed=读取证书信息失败
 license.verify.failed=验证证书失败
 license.verify.server-id.unmatch=hugegraph-hubble 服务的 id {0} 不在已授权范围内
 license.verify.version.unmatch=hugegraph-hubble 的当前版本 {0} 不在已授权范围内 <= {1}
-license.verify.graphs.exceed=图连接个数 {0} 超过证书限制 {1}
+license.verify.graphs.exceed=图连接个数 {0} 超过社区版上限 {1} 个
+license.verify.datasize.exceed=图 {0} 已占用磁盘 {1}MB 超过社区版上限 {2}MB
```

Comment:
add one more param for edition

Reply:
The message key nested is too complicated, I think remove edition is better